lib/dpkg.sh

```diff
@@ -132,7 +132,18 @@ dpkg_Sy() {
 }
 
 dpkg_Ss() {
-  apt-cache search "$@"
+  IFS=$'\n' packages=($(apt-cache search "$1"))
+  for package in ${packages[@]}
```

Comment:
Can you please change `${packages[@]}` to `${packages[@]:-}`. Without this change you see an error: `/usr/bin/pacman: line 989: packages[@]: unbound variable`. Thanks a lot.

Reply:
(I think that's a bash-4.3 (ubuntu-14.04) issue)

Reply:
I create new commit that check array before. Please check it